tests/fluka/test_fluka_converter.py

```diff
@@ -7,7 +8,7 @@
 * default physics settings for hadron therapy
 DEFAULTS                                                              HADROTHE
 * beam source
-BEAM           -0.06       0.0       0.0      -2.0      -2.0          PROTON
+BEAM      1.5000E-01                                                  PROTON
```

Comment:
That seems wrong. 
Take a look here
https://flukafiles.web.cern.ch/manual/chapters/description_input/description_options/beam.html

positive number is interpreted as momentum, negative as energy